cpp/data/system_updates/18.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -o errexit
+
+if [[ $TUEFIND_FLAVOUR == "ixtheo" ]]; then
+    if [ -e /var/spool/cron/root ]; then # CentOS
+        root_crontab=/var/spool/cron/root
+    elif [ -e /var/spool/cron/crontabs/root ]; then # Ubuntu
+        root_crontab=/var/spool/cron/crontabs/root
+    else
+        exit 0 # There is no crontab for root.
+    fi
+    sed --in-place 's/"$LOG_DIR\/generate_beacon_file_relbib.log" 2>\&1/"$LOG_DIR\/generate_beacon_file_relbib.log" 2>\&1\n15 20 * * * cd "$BSZ_DATEN" \&\& "$BIN\/generate_beacon_file.py" "--filter-field=BIB" "$EMAIL" "\/usr\/local\/ub_tools\/cpp\/data\/bibstudies-beacon.header" "$VUFIND_HOME\/public\/docs\/bibstudies-beacon.txt" > "$LOG_DIR\/generate_beacon_file_bibstudies.log" 2>\&1\n45 20 * * * cd "$BSZ_DATEN" \&\& "$BIN\/generate_beacon_file.py" "--filter-field=CAN" "$EMAIL" "\/usr\/local\/ub_tools\/cpp\/data\/canonlaw-beacon.header" "$VUFIND_HOME\/public\/docs\/canonlaw-beacon.txt" > "$LOG_DIR\/generate_beacon_file_canonlaw.log" 2>\&1/g' $root_crontab
```

Comment:
Please make sure that the order of the cronjobs will be the same as the one that the installer generated.